docs/src/examples/ode/freefall_tstop.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+# Authors: P. Kišon
+"""
+This example shows the most simple way of using a solver.
+We solve free falling of an object from height Y0 with teleportation
+happening when the object reaches height Y1, Y1 < Y0. The object
+is then teleported back to height Y0
+We use the CVODE solver to demonstrate the 'ontstop' option and
+the reinit_IC() method of the CVODE solver.
+"""
+from __future__ import print_function
+import numpy as np
+from scikits.odes.sundials import cvode
+
+#data
+g  = 9.81    # gravitational constant
+
+Y0 = 1000.0  # Initial height
+Y1 = 10.0    # Bottom height - when reached, changes to Y0 (teleport)
```

Comment:
Y1 is not used here, remove mentions to it. This freefalls up to a time at which teleportation happens, instead of up to a position. Fir doc for this